packages/fxa-content-server/server/lib/validation.js

```diff
@@ -26,7 +26,7 @@ const PATTERNS = {
   EXPERIMENT: /^[\w.-]+$/,
   FORM_TYPE: /^(email|other|button|subscribe)$/,
   PRODUCT_ID: /^prod_[0-9A-Za-z]+$/,
-  SERVICE: /^(sync|content-server|none|[0-9a-f]{16})$/,
+  SERVICE: /^(sync|content-server|none|[a-zA-Z0-9\-]{1,16})$/,
```

Comment:
I noticed the auth server allows an empty service name. This seems more correct though. Perhaps the `sync|content-server|none` are redundant?